packages/nuka/src/carousel.tsx

```diff
@@ -595,7 +600,7 @@ export const Carousel = (rawProps: CarouselProps): React.ReactElement => {
           width: '100%',
           position: 'relative',
           outline: 'none',
-          height: adaptiveHeight ? `${frameHeight}px` : 'auto',
+          height: frameHeight,
```

Comment:
Nice. I wonder if we should also add a default transition for this `height` property so that the height change isn't so sudden. Something like:

```
transition: 'height 300ms ease-in-out',
willChange: 'height',
```

I see that this can be handled manually by passing in style props, but feels like it's worth having a sane default here.

Reply:
I agree with making the transition smoother. This can be added in a follow up as well, but if we do add animations as default we should document it as well with how to disable it because it could be a surprise to users of the library.

Reply:
Good callout, Carlos! So let's leave this be for now and hit it as a follow-up task, since this PR already has a significant amount of work in it.